servicetalk-opentelemetry-http/src/main/java/io/servicetalk/opentelemetry/http/OpenTelemetryOptions.java

```diff
@@ -80,6 +89,27 @@ public boolean enableMetrics() {
         return enableMetrics;
     }
 
+    /**
+     * The {@link OpenTelemetry} instance to use for creating spans.
+     *
+     * @return the {@link OpenTelemetry} instance
+     */
+    public OpenTelemetry openTelemetry() {
+        return openTelemetry;
+    }
+
+    /**
+     * The component name used during building new spans.
+     * <p>
+     * This is a client-specific option that maps to the {@code peer.service} attribute
+     * and will be ignored when used with server filters.
+     *
+     * @return the component name
+     */
+    public String componentName() {
```

Comment:
If `componentName` is only client-specific option, it will be great if we can separate it somehow. Some options to consider:
1. Keep `componentName` as a 2nd argument for client-side filter.
2. Use `ClientOpenTelemetryOptions` and `ServerOpenTelemetryOptions`, similar to `ClientSslConfig` and `ServerSslConfig`.
3. Instead of having a builder for options, we can consider builders for filters.

Think which one you like and let's discuss tomorrow when we sync either with the team or 1-1.